docs/configuration/system/syslog.rst

```diff
@@ -17,6 +17,23 @@ Syslog supports logging to multiple targets, those targets could be a plain
 file on your VyOS installation itself, a serial console or a remote syslog
 server which is reached via :abbr:`IP (Internet Protocol)` UDP/TCP.
 
+Global
+------
+
+.. cfgcmd:: system syslog global interval <number>
```

Comment:
```suggestion
.. cfgcmd:: system syslog global marker interval <number>
```

The command is wrong, missing the `marker` option.

Reply:
Thanks, updated the commit